tekton/publish.yaml

```diff
@@ -9,55 +9,78 @@ spec:
     - name: versionTag
       description: The vX.Y.Z version that the artifacts should be tagged with (including `v`)
     - name: imageRegistry
-      description: TODO(#569) This is a hack to make it easy for folks to switch the registry being used by the many many image outputs
-    - name: pathToProject
-      description: The path to the folder in the go/src dir that contains the project, which is used by `ko` to name the resulting images
+      description: The target image registry
+      default: gcr.io
     - name: releaseAsLatest
       description: Whether to tag and publish this release as the latest
       default: "true"
-  resources:
-    inputs:
-      - name: dashboard-source-repo
-        type: git
-        targetPath: go/src/github.com/tektoncd/dashboard
-      - name: bucket-for-dashboard
-        type: storage
-    outputs:
-      - name: bucket-for-dashboard
-        type: storage
-      - name: builtDashboardImage
-        type: image
+    - name: package
+      description: package to release (e.g. github.com/<org>/<project>)
+      default: github.com/tektoncd/dashboard
+    - name: images
+      description: List of cmd/* paths to be published as images  in release manifest release.yaml
+      default: "dashboard"
+    - name: imageRegistryPath
+      description: The path (project) in the image registry
+    - name: imageRegistryRegions
+      description: The target image registry regions
+      default: "us eu asia"
+    - name: platforms
+      description: Platforms to publish for the images (e.g. linux/amd64,linux/arm64)
+      default: linux/amd64,linux/arm64,linux/s390x,linux/ppc64le
```

Comment:
We should include `linux/arm` too since that's included in our current releases.

e.g. from the latest nightly build:
```
2021/09/02 03:26:40 Using base gcr.io/distroless/base:nonroot for github.com/tektoncd/dashboard/cmd/dashboard
2021/09/02 03:26:42 Building github.com/tektoncd/dashboard/cmd/dashboard for linux/amd64
2021/09/02 03:27:53 Building github.com/tektoncd/dashboard/cmd/dashboard for linux/arm
2021/09/02 03:29:02 Building github.com/tektoncd/dashboard/cmd/dashboard for linux/arm64
2021/09/02 03:30:09 Building github.com/tektoncd/dashboard/cmd/dashboard for linux/ppc64le
2021/09/02 03:31:15 Building github.com/tektoncd/dashboard/cmd/dashboard for linux/s390x
```